lib/dynamic_validations.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# This had to be loaded to call constantize.
+# To call constantize on a string symbol, the class must be eagerly loaded.
+require_relative '../app/validators/nova_seq6000_validator'
```

Comment:
Suggestions for this are welcome :)

Reply:
You might be able to eagerly load the app/validators by including it in the either autoload_paths or eagerload_paths in config/application.rb